packages/ckeditor5-list/src/listproperties/ui/listpropertiesview.ts

```diff
@@ -119,10 +124,11 @@ export default class ListPropertiesView extends View {
 	 */
 	constructor(
 		locale: Locale,
-		{ enabledProperties, styleButtonViews, styleGridAriaLabel }: {
+		{ enabledProperties, styleButtonViews, styleGridAriaLabel, isStyleVisible }: {
 			enabledProperties: ListPropertiesConfig;
 			styleButtonViews: Array<ButtonView> | null;
 			styleGridAriaLabel: string;
+			isStyleVisible: boolean;
```

Comment:
This won't be needed if you pass a normalized config in the `propertiesConfig` parameter.